### PR TITLE
Translation Doc

### PR DESCRIPTION
**Pull request by axuebin**
Fixes #xx(issue number here), remove this line if no related issue.
#206

Changes you made in this pull request:

- Add English documentation for component ( Layout, Icon, Affix )
- New Documentation named "README-en_US.md"

You check it.

If you think the translation is OK, I will continue.

### CODE REVIEW

**Comment by axuebin** [comment]
请问我翻译的可以么？

我继续翻译吗？

File: packages/zent/src/affix/README-en_US.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+### API
+
+| Property           | Description                                                      | Type     | Default  | Options |
+| ------------------ | ---------------------------------------------------------------- | -------- | -------- | ------- |
+| offsetTop          | Pixels to offset from top when calculating position of scroll    | number   | 0        | ''      |
+| offsetBottom       | Pixels to offset from bottom when calculating position of scroll | number   | null     | null    |
+| onPin              | Callback when pin Affix                                          | function | null     | null    |
+| onUnpin            | Callback when unpin Affix                                        | function | null     | null    |
+| zIndex             | The z-index of Affix                                             | number   | 10       | null    |
+| className          | The name of the customized additional class                      | string   | `''`     |         |
```
Comment (fairyaierl) [inline packages/zent/src/affix/README-en_US.md]: custom class name

File: packages/zent/src/affix/README-en_US.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+| placeHoldClassName | The class name of container                                      | string   | `''`     |         |
+| prefix             | The customized prefix                                            | string   | `'zent'` |         |
```
Comment (fairyaierl) [inline packages/zent/src/affix/README-en_US.md]: custom prefix

File: packages/zent/src/alert/README-en_US.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
+## Alert 
+
+Alert, providing clear prompt messages.
```
Comment (fairyaierl) [inline packages/zent/src/alert/README-en_US.md]: Alert provides a striking prompt message.

File: packages/zent/src/alert/README-en_US.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+:::
+
+:::demo The conent can be non-strings such as a DOM element.
+```jsx
+import { Alert, Icon } from 'zent';
+
+ReactDOM.render(
+	<Alert>
+		<Icon type="error-circle" />
+		<span>Warning：SMS notification during the transaction will be sent via the "Message Push" function of the Marketing Center.</span>
```
Comment (fairyaierl) [inline packages/zent/src/alert/README-en_US.md]: SMS notification of trading will be sent via the "Message Push" function of the Marketing Center.

File: packages/zent/src/alert/README-en_US.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+:::demo Two sizes
+```jsx
+import { Alert } from 'zent';
+
+ReactDOM.render(
+	<div className="zent-alert-example">
+		<Alert>This is the normal size.</Alert>
+		<Alert size="large">
+			<p>This is the big size</p>
+			<p>SMS notification during the transaction will be sent via the "Message Push" function of the Marketing Center.</p><br />
```
Comment (fairyaierl) [inline packages/zent/src/alert/README-en_US.md]: SMS notification of trading will be sent via the "Message Push" function of the Marketing Center.

File: packages/zent/src/alert/README-en_US.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+:::
+
+:::demo Close button
+```jsx
+import { Alert, Button } from 'zent';
+
+ReactDOM.render(
+	<Alert size="large" closable>
+		<div className="content">
+			<p className="text">SMS notification during the transaction will be sent via the "Message Push" function of the Marketing Center.</p>
```
Comment (fairyaierl) [inline packages/zent/src/alert/README-en_US.md]: SMS notification of trading will be sent via the "Message Push" function of the Marketing Center.

File: packages/zent/src/alert/README-en_US.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+```
+:::
+
+### API
+
+| Property  | Description                                 | Type   | Default    | Options                           |
+| --------- | ------------------------------------------- | ------ | ---------- | --------------------------------- |
+| type      | Type of Alert styles                        | string | `'info'`   | `'info'`, `'warning'`, `'danger'` |
+| size      | Size of Alert                               | string | `'normal'` | `'normal'`, `'large'`             |
+| rounded   | Whether rounded                             | bool   | `false`    | `true`, `false`                   |
```
Comment (fairyaierl) [inline packages/zent/src/alert/README-en_US.md]: Whether Alert is rounded

File: packages/zent/src/alert/README-en_US.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+| closable  | Whether Alert can be closed                 | bool   | `false`    | `true`, `false`                   |
+| onClose   | Callback when close Alert                   | func   | `noop`     |                                   |
+| className | The name of the customized additional class | string | `''`       |                                   |
```
Comment (fairyaierl) [inline packages/zent/src/alert/README-en_US.md]: the custom class name

File: packages/zent/src/alert/README-en_US.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+| prefix    | The customized prefix                       | string | `'zent'`   |                                   |
+
+
+<style>
```
Comment (fairyaierl) [inline packages/zent/src/alert/README-en_US.md]: the custom prefix

File: packages/zent/src/badge/README-en_US.md [hunk 10]
```diff
@@ -0,0 +1,162 @@
+## Badge
+
+Badge normally appears in proximity to notification or head picture with eye-catching appeal, typically displaying unread messages count.
```
Comment (fairyaierl) [inline packages/zent/src/badge/README-en_US.md]: Badge normally appears in the upper right corner of the notification or avatar with eye-catching appeal, typically displaying the count of unread messages.

File: packages/zent/src/badge/README-en_US.md [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+
+### Usage guide
+
+-  Used to prompt for new messages，placed in the upper right corner or right side of the text or icon.
```
Comment (fairyaierl) [inline packages/zent/src/badge/README-en_US.md]: Badge is used to prompt for new messages，which is placed in the upper right corner or right side of the text or icon

File: packages/zent/src/badge/README-en_US.md [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+ReactDOM.render(
+	<Badge count={5}>
+		<Icon type="bell-o" className="demo-cont"/>
+	</Badge>
+	, mountNode
+);
+```
+:::
+
+:::demo Max messages count
```
Comment (fairyaierl) [inline packages/zent/src/badge/README-en_US.md]: Set the max count of messages to show

File: packages/zent/src/badge/README-en_US.md [hunk 13]
```diff
@@ -0,0 +1,162 @@
...
+
+	
+}
+ReactDOM.render(
+	<Demo />, mountNode
+);
+```
+:::
+
+:::demo Red badge without number
```
Comment (fairyaierl) [inline packages/zent/src/badge/README-en_US.md]: Red dot without number

File: packages/zent/src/badge/README-en_US.md [hunk 14]
```diff
@@ -0,0 +1,162 @@
...
+
+### API
+
+| Property  | Description                                 | Type     | Default      | Options          |
+| ----------| ------------------------------------------- | ------   | ------------ | ---------------- |
+| count     | Messages count to show                      | int      | `0`          |                  |
+| maxCount  | Max count to show                           | int      | `99`         |                  |
+| dot       | Whether to show red dot without number      | bool     | `false`      | `true`,`false`   |
+| showZero  | Whether to show badge when count is zero    | bool     | `false`      | `true`,`false`   |
+| className | The name of the customized additional class | string   | `''`         |                  |
```
Comment (fairyaierl) [inline packages/zent/src/badge/README-en_US.md]: custom class name

File: packages/zent/src/badge/README-en_US.md [hunk 15]
```diff
@@ -0,0 +1,162 @@
...
+| prefix    | The customized prefix                       | string   | `'zent'`     |                  |
```
Comment (fairyaierl) [inline packages/zent/src/badge/README-en_US.md]: custom prefix

File: packages/zent/src/icon/README-en_US.md [hunk 16]
```diff
@@ -0,0 +1,371 @@
+## Icon
+
+Semantic icon library.
+
+### How to use
+
+-   Some icons are divided into two versions:solid and scanning line which distinguished by -o(letter), e.g. `shop` (Solid) amd `shop-o` (Scanning line).
```
Comment (fairyaierl) [inline packages/zent/src/icon/README-en_US.md]: solid and scanning line which is distinguished by -o(letter)

File: packages/zent/src/icon/README-en_US.md [hunk 17]
```diff
@@ -0,0 +1,371 @@
...
+-   Naming rules: `icon name-[variant(shape)]-[scanning line?(o)]`.
```
Comment (fairyaierl) [inline packages/zent/src/icon/README-en_US.md]: icon name-[variant]-[scanning line?(o)]

File: packages/zent/src/icon/README-en_US.md [hunk 18]
```diff
@@ -0,0 +1,371 @@
...
+);
+```
+:::
+
+### API
+
+| Property  | Description                                 | Type   | Default |
+| --------- | ------------------------------------------- | ------ | ------- |
+| type      | Type of zent icons                          | string | `''`    |
+| className | The name of the customized additional class | string | `''`    |
```
Comment (fairyaierl) [inline packages/zent/src/icon/README-en_US.md]: custom class name

File: packages/zent/src/icon/README-en_US.md [hunk 19]
```diff
@@ -0,0 +1,371 @@
...
+}
+
+.zi-grid {
+    display: flex;
+    flex-wrap: wrap;
+
+    .zenticon {
+        vertical-align: middle;
+        font-size: 20px;
+		color: #333;
```
Comment (fairyaierl) [inline packages/zent/src/icon/README-en_US.md]: 缩进多了

File: packages/zent/src/layout/README-en_US.md [hunk 20]
```diff
@@ -0,0 +1,72 @@
...
+	, mountNode
+);
+```
+:::
+
+### Row API
+
+| Property  | Description                                 | Type   | Default  |
+| --------- | ------------------------------------------- | ------ | -------- |
+| className | The name of the customized additional class | string |          |
```
Comment (fairyaierl) [inline packages/zent/src/layout/README-en_US.md]: custom class name

File: packages/zent/src/layout/README-en_US.md [hunk 21]
```diff
@@ -0,0 +1,72 @@
...
+| prefix    | UI prefix                                   | string | `'zent'` |
+
+### Col API
+
+| Property  | Description                                         | Type   | Default  |
+| --------- | --------------------------------------------------- | ------ | -------- |
+| span      | The number of cells column occupied                 | number |          |
```
Comment (fairyaierl) [inline packages/zent/src/layout/README-en_US.md]: The cell number of the column occupied

File: packages/zent/src/layout/README-en_US.md [hunk 22]
```diff
@@ -0,0 +1,72 @@
...
+| offset    | The number of cells to the left of the grid spacing | number |          |
```
Comment (fairyaierl) [inline packages/zent/src/layout/README-en_US.md]: the cell number of the column offset from left

File: packages/zent/src/layout/README-en_US.md [hunk 23]
```diff
@@ -0,0 +1,72 @@
...
+| className | The name of the customized additional class         | string |          |
```
Comment (fairyaierl) [inline packages/zent/src/layout/README-en_US.md]: custom class name

File: packages/zent/src/affix/README-en_US.md (hunk 1 above)
Comment (axuebin) [inline packages/zent/src/affix/README-en_US.md]: 这个以后就都这样翻译是吗。。我那时候确实也拿不准。。

File: packages/zent/src/icon/README-en_US.md (hunk 16 above)
Comment (axuebin) [inline packages/zent/src/icon/README-en_US.md]: 啊~ 漏了~

**Comment by LucasIcarus** [comment]
i18n 这样做效率太低。。 最好能在 loader 层面操作把 demo 独立出来，减少复制粘贴的部分。

**Comment by axuebin** [comment]
好，我了解一下。

**Pull request closed without merging**